src/skeletal_model.h

```diff
@@ -78,12 +78,12 @@ typedef struct ss_bone_tag_s
 
     float                   qrotate[4];                                         // quaternion rotation
     
-#ifdef _WIN64
-	float                   local_transform[16];       // 4x4 OpenGL matrix for stack usage
-	float                   current_transform[16];    // 4x4 OpenGL matrix for global usage
-#elif __linux__
-	float                   local_transform[16] __attribute__((packed, aligned(16)));       // 4x4 OpenGL matrix for stack usage
-	float                   current_transform[16]  __attribute__((packed, aligned(16)));    // 4x4 OpenGL matrix for global usage
+#ifdef __GNUC__
```

Comment:
more correct to check the compiler, because I use GCC under Windows, so It is possible to have __attribute__ under MSW